nethermind/Dockerfile

```
@@ -0,0 +1,49 @@
+FROM golang:1.21 AS op
+
+WORKDIR /app
+
+ENV REPO=https://github.com/ethereum-optimism/optimism.git
+ENV VERSION=v1.9.1
+ENV COMMIT=4797ddb70e05d4952685bad53e608cb5606284e6
+RUN git clone $REPO --branch op-node/$VERSION --single-branch . && \
+    git switch -c branch-$VERSION && \
+    bash -c '[ "$(git rev-parse HEAD)" = "$COMMIT" ]'
+
+RUN cd op-node && \
+    make VERSION=$VERSION op-node
+
+FROM mcr.microsoft.com/dotnet/sdk:8.0-noble AS build
+
+ARG BUILD_CONFIG=release
+ARG TARGETARCH
+
+WORKDIR /app
+
+ENV REPO=https://github.com/NethermindEth/nethermind.git
+ENV VERSION=1.29.0
+ENV COMMIT=e324dae2a92e23f2cd351d81e9e46554591b2cef
+RUN git clone $REPO --branch release/$VERSION --single-branch . && \
+    git switch -c release/$VERSION && \
```

to fix Docker build:
```suggestion
    git switch -c branch-$VERSION && \
```

---

updated